js/controller.js

```diff
@@ -1464,6 +1464,7 @@ module.exports = function(OO, _, $, W) {
         this.state.adEndTime = this.state.adStartTime + this.state.adVideoDuration;
         this.skin.state.currentPlayhead = 0;
         this.removeBlur();
+        this.renderSkin();
```

Comment:
Is it not automatically rerendered when changing states?

Reply:
nope
`state` here is just mutable class field

Reply:
@KseniiaSokolovaEpam setState method doing what you expect. But it's not used here :)

Reply:
@ooyala-amohnatkin @zelibobla ok, thank you, guys!